install/kubernetes/cilium/templates/validate.yaml

```diff
@@ -153,7 +153,7 @@
 
 {{/* validate clustermesh-apiserver */}}
 {{- if .Values.clustermesh.useAPIServer }}
-  {{- if and (ne .Values.identityAllocationMode "crd") (ne .Values.identityAllocationMode "doublewrite-readkvstore") (ne .Values.identityAllocationMode "doublewrite-readcrd") }}
+  {{- if and (ne .Values.identityAllocationMode "crd") (ne .Values.identityAllocationMode "doublewrite-readkvstore") (ne .Values.identityAllocationMode "doublewrite-readcrd") (not .Values.clustermesh.kvstoremeshExternalKVStore) }}
     {{ fail (printf "The clustermesh-apiserver cannot be enabled in combination with .Values.identityAllocationMode=%s. To establish a Cluster Mesh, directly configure the parameters to access the remote kvstore through .Values.clustermesh.config" .Values.identityAllocationMode ) }}
   {{- end }}
   {{- if .Values.disableEndpointCRD }}
```

Comment:
The `disableEndpointCRD` validation can be skipped if `kvstoremeshExternalKVStore=true`.

Reply:
I believe this comment has already been resolved: https://github.com/cilium/cilium/pull/34174/files#diff-63d6e1fdb929a5058c43790c57f422863b269e45c2be10eb6564ca51fe1a0f4eR155